benchmarks/api_benchmark.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Microbenchmarks for JAX `api` functions."""
+import functools
+import operator
+
+import jax
+import jax.numpy as jnp
+
+import benchmark
```

Comment:
Can you add an inline comment here specifying what the `benchmark` import refers to?

Reply:
I'm actually surprised this works. When I tried this myself (not this PR, I was experimenting with converting pmap_benchmark.py), it would pick up the benchmark.py file in this directory, not the google-benchmark module. Did you verify this works outside your CL?

If not, I can post a PR for pmap_benchmark that gets rid of the local benchmark.py. Or maybe there's some Python-foo that lets you import the right thing.

Reply:
I would strongly suggest revising the module name, e.g., using `import google_benchmark` or maybe something more generic like `import gbenchmark`.

By default, the assumption is that `import benchmark` is referring to the project of that name registered on pypi: https://pypi.org/project/benchmark

Reply:
Okay. I'm updating the PyPi package, so the package name is `google_benchmark`.